Source/Utils/Source/Event/NativeEventDispatcher.h

```diff
@@ -50,7 +50,7 @@ class NativeEventDispatcher
 	typedef std::unordered_map< std::string, std::vector< EventHandle > > EventIdToHandleMapType;
 
 	EventHandleToContextMapType	EventHandleToContextMap;
-	EventIdToHandleMapType EventIdToHandleMap;
+	EventIdToHandleMapType EventIdToListenerHandlesMap;
```

Comment:
Oh, it didn't really occur to me earlier, but do we want to change the typedef too?

Reply:
I'm starting to think that "ListenerHandles" is maybe not the best terminology because it doesn't align with the actual types. I think I might change it to `EventIdToHandlesMap` instead.

Reply:
Yeah, I was thinking that as I was trying to figure out what it the typedef name would be. :P  I think this is fine now!